### DISCUSSION TITLE
Circular Land Tool

### DISCUSSION THREAD

**Discussion by dr-dirt** [OpenRCT2/OpenRCT2#15741] [Feature requests] [answered]
### Describe the feature

I suggest adding an option for the land tool to be used as a circle rather than a square.  

It is difficult to put perfect circles in the game at present when changing land or placing scenery.

I propose an option to select the shape of the land tool (square or circle) as well as size (length or diameter) to easily/quickly change land height/color as a perfect circle.  This has a lot of practicality for laying or circular paths, plazas, mountains, etc.

### Feature type

Quality of life

**Comment by duncanspumpkin** [comment]
Surprisingly difficult to do this neatly.

**Comment by J-PIE-314** [comment, marked as answer]
This feature has been implemented (among other amazing features) in this new plug-in for OpenRCT2: [worldpainter](https://github.com/Sadret/openrct2-worldpainter)
